Qase_Test/Src/Pages/Project/ProjectPage.cs

```diff
@@ -0,0 +1,30 @@
+using OpenQA.Selenium;
+using Qase_Test.Core.Browser.Service;
+using Qase_Test.Pages.Base;
+
+namespace Qase_Test.Pages.Project
+{
+    public class ProjectPage : BasePage
+    {
+        private const string TitleSelector = "//p[@class='header']";
+        private static readonly By ProjectPageSelector = By.XPath("//a[@data-bs-original-title='Settings']");
+        private readonly By _settingsSelector = By.XPath("//a[@data-bs-original-title='Settings']");
+        private readonly By _projectsButtonSelector = By.XPath("//a[@aria-label='Projects']");
+
+        public ProjectPage() : base(ProjectPageSelector)
+        {
+        }
+
+        private static IWebElement GetElement(By locator) =>
```

Comment:
get rid of duplication of the implementation of this method; move the implementation of the method to the base class of the page.